src/qasm.jl

```diff
@@ -12,9 +12,10 @@ const qasm_map =
          "cnot" => "CNOT",
          "swap" => "SWAP",
          "s" => "S",
-         "sdg" => "S", # "S_Dagger", # Temporary Hack!
+         "sdg" => "S_DAG",
          "t" => "T",
          "tdg" => "T", # "T_Dagger", # Temporary Hack!
```

Comment:
What is preventing us from implementing T dagger as a gate. It won't be supported by stim, but we can just throw a warning and repace it with T or something?